src/components/autocomplete/js/autocompleteController.js

```diff
@@ -274,9 +274,11 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         $scope.searchText = val;
         handleSelectedItemChange(selectedItem, previousSelectedItem);
       });
+    } else {
```

Comment:
This appears unrelated to the description of the commit/PR. Can you briefly tell us what this is for? If it's a separate fix, that's okay, we just need to update the commit message to add a mention about it :-)